Strassens_Algorithm/Strassens_Algorithm.py

```diff
@@ -0,0 +1,87 @@
+#Strassen's matrix multiplication Algorithm for NxN matrix
+
+#Function to form sub matrices
+def split_matrix(a):
+    matrix_length = len(a)
+    mid = matrix_length // 2
+    A00 = [[a[i][j] for j in range(mid)] for i in range(mid)]
+    A10 = [[a[i][j] for j in range(mid)] for i in range(mid, matrix_length)]
+    A01 = [[a[i][j] for j in range(mid, matrix_length)] for i in range(mid)]
+    A11 = [[a[i][j] for j in range(mid, matrix_length)] for i in range(mid, matrix_length)]
+    return A00 , A01,A10,A11
```

Comment:
please remove spaces around ,

Reply:
Please resolve this also.

Reply:
can you please specify ?

Reply:
A00 , A01,A10,A11 should be A00,A01,A10,A11

Reply:
You have to remove spaces around comma here